lib/open_api_spex/cast/error.ex

```diff
@@ -321,6 +337,14 @@ defmodule OpenApiSpex.Cast.Error do
     "Missing field: #{name}"
   end
 
+  def message(%{reason: :missing_header, name: name}) do
+    "Missing header: #{name}"
+  end
+
+  def message(%{reason: :invalid_header, name: name, value: value}) do
+    "Invalid value for header: #{name}. Got: #{inspect(value)}"
```

Comment:
We've been trying to avoid including input values in the error message, to eliminate the possibility of DOS attack the library could open. But I don't know. I haven't done any analysis of this attack vector. What do you think?

Reply:
👍 headers can be up to 4096 length, better not to dump them out to logs etc.